examples/simple.rs

```diff
@@ -243,9 +244,10 @@ impl From<InodeAttributes> for fuser::FileAttr {
 
 // Stores inode metadata data in "$data_dir/inodes" and file contents in "$data_dir/contents"
 // Directory data is stored in the file's contents, as a serialized DirectoryDescriptor
+#[derive(Clone)]
 struct SimpleFS {
     data_dir: String,
-    next_file_handle: AtomicU64,
+    next_file_handle: Arc<AtomicU64>,
```

Comment:
For example, `Arc<AtomicU64>` is a totally reasonable way to handle mutability, here - but this example could have easily used a normal `u64` before these changes and still derived `Clone`, leading to coinciding u64s.